.github/workflows/velox_backend_x86.yml

```diff
@@ -610,7 +610,7 @@ jobs:
             --extra-conf=spark.celeborn.client.eagerlyCreateInputStream.threads=4 \
             --benchmark-type=ds --error-on-memleak \
             --off-heap-size=16g -s=1.0 --threads=16 --iterations=1
-          if [ "${{ matrix.writer }}" = "sort" ]; then
+          if false; then # TODO: fix celeborn rss sort issue
             GLUTEN_IT_JVM_ARGS=-Xmx16G sbin/gluten-it.sh queries-compare \
```

Comment:
@kerwin-zk ran into similar issue as https://github.com/apache/incubator-gluten/pull/11105 
it seems when running q72 the worker starts to fail, is this also due to the disk space issue? 
https://github.com/apache/incubator-gluten/actions/runs/19638873977/job/56279716156